src/modules/connections-tree/items/serveritem.cpp

```diff
@@ -65,6 +65,20 @@ int ServerItem::row() const
     return TreeItem::row();
 }
 
+QVariantMap ServerItem::getFilterHistory() {

```

Comment:
This methods should be placed in DatabaseItem

Reply:
Not sure about it, my idea was to put m_filterHistory into ServerItem so the same list would be available for every DatabaseItem and saved to connections.json as a plain object (`"filter_history": { "*": 1 }`) without nested db0-db15.

Also then I won't be able use `getFilterHistory()` in `DatabaseItem::filterHistoryTop10()` and `DatabaseItem::metadata()` and so on as they are `const`, but `getFilterHistory()` modifies `m_filterHistory` in `DatabaseItem` object to get history from config, so it can't be `const`.

I'll think it over again.